api/python/provisioner/commands/deploy.py

```diff
@@ -312,14 +330,18 @@ def _run_states(self, states_group: str, run_args: run_args_type):
                     self._apply_state(
                         f"components.{state}", secondaries, stages
                     )
-                elif state in ("ha.cortx-ha"):
-                    # Execute first on primary then on secondaries in a sequence.
-                    self.ensure_consul_running()
```

Comment:
why that was removed?

Reply:
`check_consul_running` was specially written for dual node and It has old consul service names now consul service names are changed. That's why I have removed. If we required this check we can add this check in separate PR.